### ISSUE TITLE
Support for Phi-2

### ISSUE THREAD

**Issue by nndnnv** [SciSharp/LLamaSharp#381]
Tried running phi-2 model, got this error:
`error loading model: unknown model architecture: 'phi2'
`

llama.cpp already added support for Phi-2 model, so I guess a new LLamaSharp version (with updated llama.cpp binary) should add support as well, correct?

[https://github.com/ggerganov/llama.cpp/pull/4490](url)

**Comment by martindevans** [comment]
Yep we'll need to update the binaries to get support for Phi2. Latest update was last week (https://github.com/SciSharp/LLamaSharp/pull/361) (not yet released). So we'll probably pickup Phi2 support in another month or so at the next update.

**Comment by martindevans** [comment]
Should be resolved in #479
